deployments/ansible/molecule/custom_vars/windows-converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Converge scenario with custom variables
+  hosts: all
+  become: no
+  vars:
+    splunk_access_token: fake-token
+    splunk_ingest_url: https://fake-ingest.com
```

Comment:
We may want to standardize on dummy targets instead of potentially spamming random sites. fake-api.com is apparently an actual service.

Reply:
Renamed to `fake-splunk-api.com` and `fake-splunk-ingest.com`.